ppb/systems/renderer.py

```diff
@@ -311,7 +311,18 @@ def compute_rectangles(self, texture, game_object, camera):
 
         win_w, win_h = self.target_resolution(img_w.value, img_h.value, obj_w, obj_h, camera.pixel_ratio)
 
-        center = camera.translate_point_to_screen(game_object.position)
+        try:
+            center = camera.translate_point_to_screen(game_object.position)
+        except TypeError as error:
+            raise TypeError(f"""{type(game_object).__name__}.position was set to a tuple: 
+
+            (number, number)
+
+            Expected a vector:
+
+            Vector(number, number)
+            """)
```

Comment:
```suggestion
            """) from error
```

Forgot this while we were pairing, this will also show users that the error originated in the camera.